cmd/tidb-lightning/main.go

```diff
@@ -74,11 +75,18 @@ func main() {
 		return
 	}
 
-	if cfg.App.ServerMode {
-		err = app.RunServer()
-	} else {
-		err = app.RunOnce()
-	}
+	err = func() error {
```

Comment:
It could be 

```golang
                        cfg := config.NewConfig()
			err = cfg.LoadFromGlobal(globalCfg)
			if err == nil {
				err = app.RunOnce(context.Background(), cfg, nil, nil)
			}
```

But I think that's more mess since `err` is assinged many place